db/db_service.go

```diff
@@ -82,6 +92,21 @@ func (svc *dbService) CreateApp(name string, pubkey string, maxAmountSat uint64,
 			}
 		}
 
+		appWalletChildPrivKey, err := walletChildPrivKeyGeneratorFunc(uint32(app.ID))
```

Comment:
The pubkey can be passed in the original App declaration at line 72. I don't think there's any need to update the app after it's created inside the transaction

Reply:
Since the key needs `app.ID`, which AFAIK will only be generated after `tx.Create` I think we need to do the update later, right?

Reply:
Ah, true. Thanks!